CHANGELOG.md

```diff
@@ -2,22 +2,32 @@
 
 This file is used to list changes made in each version of the splunk cookbook.
 
+## Unreleased
+
+- Sous Chefs Adoption
+- Re-vamp inspec tests for test-kitchen
+- Various cookstyle & spec fixes
+
 ## 7.1.0 (2020-11-30)
+
 - Adds cookbook dependency on [ec2-tags-ohai-plugin](https://supermarket.chef.io/cookbooks/ec2-tags-ohai-plugin) to read EC2 tags as a secondary detection method for rotating Splunk secrets
 
 ## 7.0.3 (2020-11-02)
+
 - Fixes Issue #128 logic in `#shcluster_members?` helper method with a better match
 
 ## 7.0.2 (2020-10-30)
+
 - Fix centos-6 and Amazon linux convergence by adding an only_if to only create systemd symlinks on systemd systems.
 - Adds clarification text to README.md regarding chef-vault fallback to encrypted data bags
 - Changes the build status badge to track Actions status
 
 ## 7.0.1 (2020-10-30)
+
 - Moves most tests back to dokken and only run suites that change splunk user from root in vagrant.
 
 ## 7.0.0 (2020-10-22)
-**BREAKING CHANGE**
```

Comment:
Keep the `BREAKING CHANGE` change, but correct the formatting.

Reply:
What is the correct formatting here? should this be part of the list or another header?

Reply:
There needs to be a new line between the header and the text.